Changelog for multirole so far (#6979)
mesosphere/marathon: changelog.md

## Changes from 1.8.212 to 1.9.xxx

### Multirole support

Marathon 1.9 brings support for multirole, enabling you to launch services for different roles (against different Mesos quotas) with the same Marathon instance.

#### Role field added to services

> The role field can now be optionally specified for a service. However, the value of this field may only be sent to 1 of two values:

Reviewer: ```suggestion
The role field can now be optionally specified for a service. However, the value of this field may only be sent to one of two values:
```

> The role field can now be optionally specified for a service. However, the value of this field may only be sent to one of two values:

Reviewer 2: ```suggestion
The role field can now be optionally specified for a service. However, the value of this field may only be set to one of two values:
```

> * The default role as specified by `--mesos_role c`ommand line parameter

Reviewer: ```suggestion
* The default role as specified by `--mesos_role` command line parameter
```
